OneWelcomeExampleApp/Interactors/ProfileInteractor.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright © 2022 OneWelcome. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
to be removed

Reply:
What do you want to remove?

Reply:
That what we always remove after creating a new file

Reply:
I've seen this on other files in Example app so I thought this is what we do, for some reason like being public code, in Example app. Anyway, I will remove it.

Reply:
sorry my bad, I forgot that this is a public repo. please leave it as it is